Client/WarFare/UICmd.cpp

```diff
@@ -56,31 +56,31 @@ CUICmd::~CUICmd()
-
-	m_pBtn_Act =			GetChildByID("btn_control");	//__ASSERT(m_pBtn_Act, "NULL UI Component!!");
-	m_pBtn_Act_Walk =		GetChildByID("btn_walk");		//__ASSERT(m_pBtn_Act_Walk, "NULL UI Component!!");
-	m_pBtn_Act_Run =		GetChildByID("btn_run");		//__ASSERT(m_pBtn_Act_Run, "NULL UI Component!!");
-	m_pBtn_Act_Stop =		GetChildByID("btn_stop");		//__ASSERT(m_pBtn_Act_Stop, "NULL UI Component!!");
-	m_pBtn_Act_Attack =		GetChildByID("btn_attack");		//__ASSERT(m_pBtn_Act_Attack, "NULL UI Component!!");
-	m_pBtn_Act_StandUp =	GetChildByID("btn_stand");		//__ASSERT(m_pBtn_Act_StandUp, "NULL UI Component!!");
-	m_pBtn_Act_SitDown =	GetChildByID("btn_sit");		//__ASSERT(m_pBtn_Act_SitDown, "NULL UI Component!!");
-
-	if(m_pBtn_Act_StandUp) m_pBtn_Act_StandUp->SetVisible(false); // 일어서기 버튼은 미리 죽여놓는다..
 	
-	m_pBtn_Character =		GetChildByID("btn_character");	//__ASSERT(m_pBtn_Character, "NULL UI Component!!");
-	m_pBtn_Inventory =		GetChildByID("btn_inventory");	//__ASSERT(m_pBtn_Inventory, "NULL UI Component!!");
-
-	m_pBtn_CmdList =			GetChildByID("btn_option");		//__ASSERT(m_pBtn_Option, "NULL UI Component!!");
-	m_pBtn_Camera =			GetChildByID("btn_camera");		//__ASSERT(m_pBtn_Camera, "NULL UI Component!!");
-	m_pBtn_Party_Invite =	GetChildByID("btn_invite");		//__ASSERT(m_pBtn_Party_Invite, "NULL UI Component!!");
-	m_pBtn_Party_Disband =	GetChildByID("btn_disband");	//__ASSERT(m_pBtn_Party_Disband, "NULL UI Component!!");
-	m_pBtn_Skill =			GetChildByID("btn_skill");		//__ASSERT(m_pBtn_Skill, "NULL UI Component!!");
-	m_pBtn_Exit	=			GetChildByID("btn_Exit");		//__ASSERT(m_pBtn_Exit, "NULL UI Component!!");
-
-	m_pBtn_Quest =		GetChildByID("btn_quest");			//__ASSERT(m_pBtn_Quest, "NULL UI Component!!");
-	m_pBtn_Belong =		GetChildByID("btn_knight");			//__ASSERT(m_pBtn_Belong, "NULL UI Component!!");
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act, GetChildByID("btn_control"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act_Walk, GetChildByID("btn_walk"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act_Run, GetChildByID("btn_run"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act_Stop, GetChildByID("btn_stop"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act_Attack, GetChildByID("btn_attack"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act_StandUp, GetChildByID("btn_stand"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Act_SitDown, GetChildByID("btn_sit"));
+
+	// 일어서기 버튼은 미리 죽여놓는다..
+	if(m_pBtn_Act_StandUp) 
+		m_pBtn_Act_StandUp->SetVisible(false); 
+	
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Character, GetChildByID("btn_character"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Inventory, GetChildByID("btn_inventory"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_CmdList, GetChildByID("btn_option"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Camera, GetChildByID("btn_camera"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Party_Invite, GetChildByID("btn_invite"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Party_Disband, GetChildByID("btn_disband"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Skill, GetChildByID("btn_skill"));
+	N3_VERIFY_UI_COMPONENT(m_pBtn_Exit, GetChildByID("btn_Exit"));
```

Comment:
This is just a note while I'm checking these in the UI, but this should just be btn_exit (it's how it's named in the UI itself). The comparison is case-insensitive so it doesn't matter, but if we were to speed this up in future with a case-sensitive lookup, this would need to be changed.

Might as well fix that one now.